primitives/src/market.rs

```diff
@@ -174,13 +176,22 @@ pub enum MarketCreation {
     Advised,
 }
 
+// TODO to remove, when Disputes storage item is removed
 #[derive(Clone, Decode, Encode, Eq, MaxEncodedLen, PartialEq, RuntimeDebug, TypeInfo)]
-pub struct MarketDispute<AccountId, BlockNumber> {
+pub struct OldMarketDispute<AccountId, BlockNumber> {
     pub at: BlockNumber,
     pub by: AccountId,
     pub outcome: OutcomeReport,
 }
 
+#[derive(Clone, Decode, Encode, Eq, MaxEncodedLen, PartialEq, RuntimeDebug, TypeInfo)]
+pub struct MarketDispute<AccountId, BlockNumber, Balance> {
+    pub at: BlockNumber,
+    pub by: AccountId,
+    pub outcome: OutcomeReport,
+    pub bond: Balance,
```

Comment:
You might mean a double booking to the market storage `MarketBonds`. The structure of `MarketDispute` is used only for simple disputes. Each dispute in simple-disputes has its own bond amount (according to `default_outcome_bond(n)`). This dispute bond does not really fit into the market structure, since it's specific to simple-disputes.

Reply:
Oh! I see. Can we move `MarketDispute` into `simple-disputes` then?

Reply:
So you mean just moving the [struct `MarketDispute`](https://github.com/zeitgeistpm/zeitgeist/blob/88a43dd06b10656a46cee4ad02b560eb9090e50d/primitives/src/market.rs#L179-L193) to the simple-disputes pallet. `MarketDispute` is currently used by court too in the code.

Reply:
Ok, in that case it should remain in primitives.